test_cli.py

```diff
@@ -2171,6 +2172,30 @@ def test(ctx):
     check_equals(_run_command([cl, 'uinfo', user_name, '-f', 'has_access']), 'True')
 
 
+@TestModule.register('edit_arbitrary_field')
```

Comment:
Thoughts on making this into a general edit test while we're at it?

Reply:
Yes, can you rename it to `edit` instead? We want our test suites in test_cli.py to be named not too specifically -- it's better to be general.